integration-tests/feature_list.txt

```diff
@@ -1,19 +1,20 @@
 # Order of the features in this file is significant, so shuffle them carefully.
-features/smoketest.feature
-features/server_api.feature
-features/jobs_api.feature
-features/jobs_debug_api.feature
+#features/smoketest.feature
+#features/server_api.feature
+#features/jobs_api.feature
+#features/jobs_debug_api.feature
 #features/ecosystems.feature
-features/components.feature
-features/stack_analyses_v2_minimal.feature
-features/user_feedback.feature
+#features/components.feature
+#features/stack_analyses_v2_minimal.feature
+#features/user_feedback.feature
 #features/versions.feature
 #features/anitya.feature
 #features/stack_analyses_v2.feature
-features/selfcheck.feature
+#features/selfcheck.feature
 #features/regression_tests.feature
-features/user_tag.feature
-features/gremlin.feature
+#features/user_tag.feature
+#features/gremlin.feature
+features/three_scale.feature
 
```

Comment:
@GeetikaBatra why are all existing features commented out? All features must run during integration test along with the one that you've added.